guides/getting-started/develop-solidus.md

```diff
@@ -0,0 +1,127 @@
+# Develop Solidus
+
+If you would like to develop for Solidus and you intend to submit your work as a
+pull request, please [read the contributing guidelines][contributing] first.
+
+Getting your Solidus development environment set up is easy. First, clone the
+Solidus GitHub repo:
+
+```shell
+git clone git://github.com/solidusio/solidus.git
+```
+
+Then enter the directory you just downloaded and install Solidus's dependencies:
+
+```shell
+cd solidus
+bundle install
+```
+
+## Create a sandbox application
+
+Solidus is meant to be run within a Rails application. You can create a sandbox
+application inside the source directory that you have cloned. This gives you a
+typical Solidus store you can use for testing.
+
+By default, the sandbox includes [`solidus_auth_devise`][solidus-auth-devise],
+and the generator seeds the database and loads sample data.
+
+```shell
+bundle exec rake sandbox
+```
+
+You can prepend `DB=mysql` or `DB=postgresql` to the command in order use those
+databases instead of the default SQLite 3 database. For example:
+
+```shell
+DB=postgresql bundle exec rake sandbox
+``` 
+
+After the sandbox has been generated, you can change into its directory and
+start the server:
+
+```
+cd sandbox
+rails server
+```
+
+[contributing]: https://github.com/solidusio/solidus/blob/master/CONTRIBUTING.md
+[solidus-auth-devise]: https://github.com/solidusio/solidus_auth_devise
+
+## Testing
+
+Solidus uses [RSpec](http://rspec.info/) for testing.
+
+If you intend to submit your work to Solidus as a pull request, it must pass all
+of the Solidus test suites before it is merged. We use CircleCI to run tests on
+all incoming pull requests.
+
+To run the test suites for `solidus_frontend` and `solidus_backend`, you need to
+install [PhantomJS](http://phantomjs.org/) on your system first.
```

Comment:
We don't use phantoms anymore.

You need to install [chromedriver](https://sites.google.com/a/chromium.org/chromedriver/home) on your system first.

Reply:
Thank you. I will also update this in the README.